cmd/cli/mesh_list.go

```diff
@@ -77,5 +71,13 @@ func (l *meshListCmd) run() error {
 	fmt.Fprint(w, getPrettyPrintedMeshInfoList(meshInfoList))
 	_ = w.Flush()
 
+	fmt.Fprintf(l.out, "\nTo list the OSM controller pods, please run the following command\n")
+	fmt.Fprintf(l.out, "\tkubectl get pods -n %s -l app=osm-controller\n", settings.Namespace())
+
+	supportedSmiVersionsEndpoint := fmt.Sprintf("http://localhost:%d%s", constants.OSMHTTPServerPort, constants.HTTPServerSmiVersionPath)
+	fmt.Fprintf(l.out, "\nTo list the supported SMI versions, please run the following command and obtain the supported versions from %s\n", supportedSmiVersionsEndpoint)
+	fmt.Fprintf(l.out, "\tkubectl port-forward -n %s $(kubectl get pods -n %s --selector app=osm-controller --no-headers | awk '{print $1}') %d &\n", settings.Namespace(), settings.Namespace(), constants.OSMHTTPServerPort)
+	fmt.Fprintf(l.out, "\tcurl %s\n", supportedSmiVersionsEndpoint)
```

Comment:
2 issues here:
1. Address https://github.com/openservicemesh/osm/pull/3677#issuecomment-869828680
2. Not use linux specific commands such as `awk` in the output. The CLI is built for windows as well.

Reply:
Resolved @shashankram. SMI info is now printed in a separate table.